doc/sjs-for-js/es6-to-scala-part3.md

```diff
@@ -260,6 +260,234 @@ Here we use triple-quoted strings that allow us to write regex without escaping
 converted into a {% scaladoc util.matching.Regex %} object with the `.r` method. Because regexes extract strings, we need
 to convert matched groups to integers ourselves.
 
+## Functions revisited
+
+We covered the basic use functions in [Part 1](es6-to-scala-part1.html), but Scala, being a functional programming
+language, provides much more when it comes to functions. Let's explore some of the more advanced features and how they
+compare to JavaScript.
+
+#### Higher-order functions
```

Comment:
OK let's keep it that way for now, but we should globally fix this another time.